ddl/index.go

```diff
@@ -1445,7 +1445,7 @@ func iterateSnapshotRows(store kv.Storage, priority int, t table.Table, version
 	ver := kv.Version{Ver: version}
 
 	snap, err := store.GetSnapshot(ver)
-	snap.SetPriority(priority)
+	snap.SetOption(kv.Priority, priority)
```

Comment:
Is this change relative to the bug?

Reply:
This change can save many lines of code.